### PR TITLE
chore(website): token list performance improvements

### PR DESCRIPTION
**Pull request by TheSisb**
- [x] Moves the copy to clipboard functionality to the token list component and only passes required, unchanged items down to TokenCard to prevent frequent list re-renders
- [x] Uses `useCallback` and `React.memo` tastefully to prevent re-renders

### CODE REVIEW

**Review by zahnster — APPROVED** [review]
Praise: Nice work! Token perf is much better (using the same test as before I'm seeing an average of 298ms to perform the change, where before I was getting 677ms).

**Review by shleewhite — APPROVED** [review]
Nice!!

File: packages/paste-website/src/components/tokens-list/token-card/index.tsx [hunk 1]
```diff
@@ -77,102 +76,106 @@ export const TokenCard: React.FC<{
...
   useCamelCase?: boolean;
-}> = ({category, name, value, comment, backgroundColor, useCamelCase}) => {
-  const tooltipState = useTooltipState();
-  const [tooltipText, setTooltipText] = React.useState('Copy token name');
-  // Prevents tooltip being visible on first render due to reakit positioning bug code
-  const isFirstRender = React.useRef(true);
-  const clipboard = useClipboard({copiedTimeout: 2000});
-  const tokenName = useCamelCase ? camelCase(name) : `$${name}`;
+  isCopied?: boolean;
+  onCopyText?: (tokenName: string) => void;
```
Comment (nkrantz) [inline packages/paste-website/src/components/tokens-list/token-card/index.tsx:80]: suggestion: should these props be abstracted into a `TokenCardProps` type?

Comment (nkrantz) [reply packages/paste-website/src/components/tokens-list/token-card/index.tsx:80]: merge went in before I finished my review!

**Pull request merged**